src/Uno.UWP/Globalization/ApplicationLanguages.cs

```diff
@@ -44,13 +44,11 @@ private static string[] GetManifestLanguages()
 			var languages = new[]
 			{
 #if __ANDROID__
-#pragma warning disable CS0618 // Type or member is obsolete
-				ContextHelper.Current.Resources.Configuration.Locale.ToLanguageTag(),
-#pragma warning restore CS0618 // Type or member is obsolete
+				ContextHelper.Current?.Resources?.Configuration?.Locales?.Get(0)?.ToLanguageTag(),
```

Comment:
not in scope for this PR but what's the current status for ContextHelper? isn't there a way to avoid it ? Like Android.Application.Context instead?  @jeromelaban

Reply:
There's no way to avoid it for now.

Reply:
Since it's not in this PR's scope, can we resolve this conversation and talk about it somewhere else ?